packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3040,8 +3040,8 @@ export default class Meeting extends StatelessWebexPlugin {
           Media.getUserMedia(
             {
               ...mediaDirection,
-              sendAudio: (mediaDirection.sendAudio === devicePermissions.sendAudio),
-              sendVideo: (mediaDirection.sendVideo === devicePermissions.sendVideo),
+              sendAudio: devicePermissions.sendAudio,
+              sendVideo: devicePermissions.sendVideo,
               isSharing: this.shareStatus === SHARE_STATUS.LOCAL_SHARE_ACTIVE
```

Comment:
Can we add a test case for camera reserve so won't break again? If the test case not there already.